lib/handlers/get.js

```diff
@@ -176,7 +176,7 @@ function globHandler (req, res, next) {
       var data = $rdf.serialize(
         undefined,
         globGraph,
-        null,
+        filename,
         'text/turtle')
```

Comment:
From what I understand, glob functionality is only spec'd/implemented for Turtle.